Odd "error[E0658]: is unstable" diagnostic

```rust
#[link(name = "c", cfg(yes))]
extern {}
```

```
error[E0658]: is unstable
 --> src/lib.rs:1:1
  |
1 | #[link(name = "c", cfg(yes))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: for more information, see https://github.com/rust-lang/rust/issues/37406
```

:thinking: 

In addition to the malformed diagnostic, the tracking issue has been closed already. According to https://rust-lang.github.io/rfcs/1721-crt-static.html#lazy-link-attributes, this is supposed to be a permanently-unstable implementation detail. In that case, should the link be updated to point somewhere else?

<!-- TRIAGEBOT_START -->

<!-- TRIAGEBOT_ASSIGN_START -->
This issue has been assigned to @Rustin-Liu via [this comment](https://github.com/rust-lang/rust/issues/70538#issuecomment-606399105).
<!-- TRIAGEBOT_ASSIGN_DATA_START$${"user":"Rustin-Liu"}$$TRIAGEBOT_ASSIGN_DATA_END -->

<!-- TRIAGEBOT_ASSIGN_END -->
<!-- TRIAGEBOT_END -->

--- a/src/librustc_metadata/native_libs.rs
+++ b/src/librustc_metadata/native_libs.rs
@@ -162,8 +162,13 @@ impl Collector<'tcx> {
             }
         }
         if lib.cfg.is_some() && !self.tcx.features().link_cfg {
-            feature_err(&self.tcx.sess.parse_sess, sym::link_cfg, span.unwrap(), "is unstable")
-                .emit();
+            feature_err(
+                &self.tcx.sess.parse_sess,
+                sym::link_cfg,
+                span.unwrap(),
+                "kind=\"link_cfg\" is unstable",
+            )
+            .emit();
         }
         if lib.kind == cstore::NativeStaticNobundle && !self.tcx.features().static_nobundle {
             feature_err(

--- a/src/test/ui/feature-gates/feature-gate-link_cfg.stderr
+++ b/src/test/ui/feature-gates/feature-gate-link_cfg.stderr
@@ -1,4 +1,4 @@
-error[E0658]: is unstable
+error[E0658]: kind="link_cfg" is unstable
   --> $DIR/feature-gate-link_cfg.rs:1:1
    |
 LL | #[link(name = "foo", cfg(foo))]

